pkg/testutil/require.go

```diff
@@ -3,30 +3,22 @@
 package testutil
 
 import (
-	"reflect"
 	"testing"
 
+	v0 "github.com/authzed/authzed-go/proto/authzed/api/v0"
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
 	"github.com/stretchr/testify/require"
 )
 
 // RequireEqualEmptyNil is a version of require.Equal, but considers nil
 // slices/maps to be equal to empty slices/maps.
 func RequireEqualEmptyNil(t *testing.T, expected, actual interface{}, msgAndArgs ...interface{}) {
-	expectedVal := reflect.ValueOf(expected)
```

Comment:
These changes should probably also be in a separate commit.

Reply:
I didn't track down exactly why this started happening, but after adding the consistency middleware some of the tests that assert with `RequireEqualEmptyNil` had some deeply nested types be unequal (because one side was nil and the other was an empty slice). I thought it was easier to make the test helper (which already collapsed those cases for top-level values) work for nested types too (and also print out more helpful error messages when they fail) instead of touching every test expectation.

TL;DR this is pretty coupled to the other changes in this commit

Reply:
That's fine, but can't you just put it in a commit before you introduce the universal consistency? universal consistency has this as a pre-requisite IIUC.